config/nrf52_pins.h

```diff
@@ -68,34 +66,30 @@
     #define SX1276_RXTX_PIN 30
 
     // RX/TX pins for the GPS UART
-    #define GPS_RX_PIN 17
-    #define GPS_TX_PIN 18
+    #define GPS_RX_PIN 29
+    #define GPS_TX_PIN 11
     #define GPS_CTS_PIN -1
     #define GPS_RTS_PIN -1
-
-    // SPI pins for the IMU
-    #define IMU_MISO_PIN 18
-    #define IMU_MOSI_PIN 17
-    #define IMU_SCK_PIN 16
-    #define IMU_CS_M_PIN 19
-    #define IMU_CS_AG_PIN 20
-
 #endif
 
 // Common pin assignments
 
-// Channel 1: GPS main power
-// Channel 2: GPS VBackup (always on)
-// Channel 3: Accelerometer/gyro
-// Channel 4: Accelerometer/gyro IO (turn on )
-#define TPS22994_GPS_PIN 1
-#define TPS22994_VBACKUP_PIN 2
-#define TPS22994_MEMS_PIN 3
-#define TPS22994_MEMS_IO_PIN 4
-
-//I2C
-#define PLAT_I2C_SDA 26
-#define PLAT_I2C_SCL 27
+// Channel 0: GPS main power
+// Channel 1: GPS VBackup (always on)
+// Channel 2: Accelerometer/gyro
+// Channel 3: Accelerometer/gyro IO (turn on )
+#define TPS22994_GPS_PIN 0
+#define TPS22994_VBACKUP_PIN 1
+#define TPS22994_BNO055_PIN 2
+#define TPS22994_BNO055_IO_PIN 3
+
+// Old EE02 I2C
```

Comment:
We still have a bunch of the previous iterations of EE02. These have the alternative SDA/SCL mapping. It would be nice to have this info as comments for the time being.